cmd/runtimetest/main.go

```diff
@@ -271,6 +271,62 @@ func validateDefaultFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateLinuxDevices(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux devices")
+
+	for _, device := range spec.Linux.Devices {
+		fi, err := os.Stat(device.Path)
+		if err != nil {
+			return err
+		}
+		fStat := fi.Sys().(*syscall.Stat_t)
```

Comment:
On 09/14/2016 11:46 PM, W. Trevor King wrote:
> This |Stat| with its type/major/minor checks (and probably the |Stat| existence check too) should get pulled out into it's own helper (|func validateDevice(path string, device *rspec.Device) error|?) so we can use it in |validateDefaultDevices| as well.

added |Stat| existence check.
I think helper function is not needed yet, as default device validation does not need so complicated check